client/src/components/PhotosModal.jsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
I'm not sure if the comma here will break anything.

Reply:
This is intentional so that I can import react and also the useState hook.